client/cmdhfmfdes.c

```diff
@@ -168,27 +168,27 @@ void getKeySettings(uint8_t *aid) {
     //memset(messStr, 0x00, 512);
 
     if (aid == NULL) {
-        PrintAndLogEx(NORMAL, " CMK - PICC, Card Master Key settings ");
-        PrintAndLogEx(NORMAL, "");
+        PrintAndLogEx(SUCCESS, "   CMK - PICC, Card Master Key settings");
+        PrintAndLogEx(INFO, "-------------------------------------------------------------");
         {
             uint8_t data[1] = {GET_KEY_SETTINGS};  // 0x45
             SendCommandOLD(CMD_HF_DESFIRE_COMMAND, INIT | DISCONNECT, sizeof(data), 0, data, sizeof(data));
         }
         if (!WaitForResponseTimeout(CMD_ACK, &resp, 1000)) {return;}
         isOK  = resp.oldarg[0] & 0xff;
         if (!isOK) {
-            PrintAndLogEx(WARNING, "   Can't select master application");
+            PrintAndLogEx(WARNING, _RED_("   Can't select master application"));
             return;
         }
 
-        str = (resp.data.asBytes[3] & (1 << 3)) ? "YES" : "NO";
-        PrintAndLogEx(NORMAL, "   [0x08] Configuration changeable       : %s", str);
-        str = (resp.data.asBytes[3] & (1 << 2)) ? "NO" : "YES";
-        PrintAndLogEx(NORMAL, "   [0x04] CMK required for create/delete : %s", str);
-        str = (resp.data.asBytes[3] & (1 << 1)) ? "NO" : "YES";
-        PrintAndLogEx(NORMAL, "   [0x02] Directory list access with CMK : %s", str);
-        str = (resp.data.asBytes[3] & (1 << 0)) ? "YES" : "NO";
-        PrintAndLogEx(NORMAL, "   [0x01] CMK is changeable              : %s", str);
+        str = (resp.data.asBytes[3] & (1 << 3)) ? _GREEN_("YES") : _RED_("NO");
```

Comment:
not need RED for no here.  
Here its important to figure out which settings is advantages for us... ie use green. 
if not,  just normal white.